gradle.properties

```diff
@@ -2,14 +2,18 @@
 # This is required to provide enough memory for the Minecraft decompilation process.
 org.gradle.jvmargs=-Xmx3G
 
-version_mc=1.12.2
-version_mod=3.62
-version_core=3.27
+# Project
+versionMC=1.12.2
+versionForge=14.23.5.2860
+versionMod=3.62
 
-version_jei=4.12.+
-version_ct=4.1.9.+
-version_rf=1.12-2.+
-version_top=1.4.+
-version_bs=2.3.+
-version_wa=1.8.26-B41_1.12.2
-version_gb=1.12.2\:2.9.+
\ No newline at end of file
+# Dependencies
+versionCore=3.27
+versionJEI=4.12.+
+versionCT=4.1.9.+
+versionRF=1.12-2.+
+versionTOP=1.4.+
+
+# Core
+coreRepo=https://github.com/slava110/Rockhounding_Core
```

Comment:
unless you're taking over the project, this shouldn't be set to your repo?

Reply:
Yep, I know, will be changed once PR will be reviewed

Reply:
hi, merged the Core PR